### DISCUSSION TITLE
Changing the .testing domain/TLD to another private domain

### DISCUSSION THREAD

**Discussion by jauyzed** [crc-org/crc#3360] [Q&A] [answered]
After a previous CRC install, I realized the dashboard will not be reachable on a private domain .testing. I followed this document: [CRC_doc](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5/html/getting_started_guide/networking_gsg#dns-configuration_gsg) and made the changes in the file(/etc/NetworkManager/dispatcher.d/99-crc.sh) to the required private domain that will be resolvable in the org but the domain name .testing is not changing after I tried to reinstall with `crc setup`. 

Changing `https://console-openshift-console.apps-crc.testing` would need to be  `https://console-openshift-console.apps-crc.<private-tld>`

Do I need to manually change this: 

`/etc/NetworkManager/dnsmasq.d/crc.conf:`

server=/crc.testing/192.168.130.11
server=/apps-crc.testing/192.168.130.11

Thank you!

**Comment by praveenkumar** [comment, marked as answer]
https://github.com/code-ready/crc/wiki/Change-the-domain-for-CRC Also if your use case is just to access registry as you created issue ~#3359~ #3361 then I don't think you need to change these settings.

**Reply by jauyzed** [reply]
@praveenkumar Thank you for sending this doc quickly. This is fairly tedious process. It would be great if this configuration can be managed via `crc setup` and/or with a `crc config set` variable

**Reply by praveenkumar** [reply]
@jauyzed Yes, we have plan around it but first we want to make sure if that works, if you tried it out and works then do let us know.
